sonaranalyzer-dotnet/rspec/cs/S4275_c#.html

```diff
@@ -0,0 +1,52 @@
+<p>Properties provide a way to enforce encapsulation by providing <code>public</code>, <code>protected</code> or <code>internal</code> methods that
+give controlled access to <code>private</code> fields. However in classes with multiple fields it is not unusual that cut and paste is used to quickly
+create the needed properties, which can result in the wrong field being accessed by a getter or setter.</p>
+<p>This rule raises an issue in any of these cases:</p>
+<ul>
+  <li> A setter does not update the field with the corresponding name. </li>
+  <li> A getter does not access the field with the corresponding name. </li>
```

Comment:
I took out the two more advanced cases so the rule has exactly the same scope as the Java implementation (i.e. doesn't check for getter/setter writing to different fields, and doesn't check if multiple properties set/return the same field)